modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,185 @@
+|type: string
+|
+
 |===
 
 
-.Configuration parameters passed to Query
+.Configuration parameters for the Query Frontend
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    options: {}
-|Configuration options that define the Query service.
+|`component`
+|TempoComponentSpec is embedded to extend this definition with further options. Currently, there is no way to inline this field.
+|type: object
+|
+
+|`nodeSelector` (inside component)
+|NodeSelector is the simplest recommended form of node selection constraint.
+|type: object, additionalProperties: type: string
+|
+
+|`replicas` (inside component)
+|Replicas represents the number of replicas to create for this component.
+|type: integer, format: int32
+|
+
+|`tolerations` (inside component)
+|Tolerations defines component specific pod tolerations.
+|type: array, x-kubernetes-list-type: atomic
+|
+
+|`Tolerations` (items, inside component)
+|The pod this Toleration is attached to tolerates any taint that matches the triple <key, value, effect> using the matching operator <operator>.
+|type: object
 |
+
+|`effect` (inside component)
+|Effect indicates the taint effect to match. Empty means match all taint effects. When specified, allowed values are NoSchedule, PreferNoSchedule, and NoExecute.
+|type: string
+|
+
+|`key` (inside component)
+|Key is the taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, operator must be Exists; this combination means to match all values and all keys.
+|type: string
+|
+
+|`operator` (inside component)
+|Operator represents a key's relationship to the value. Valid operators are Exists and Equal. Defaults to Equal. Exists is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
+|type: string
+|
+
+|`tolerationSeconds` (inside component)
+|TolerationSeconds represents the period of time the toleration (which must be of effect NoExecute, otherwise this field is ignored) tolerates the taint. By default, it is not set, which means tolerate the taint forever (do not evict). Zero and negative values will be treated as 0 (evict immediately) by the system.
+|type: integer, format: int64
 |
 
-|options:
-  log-level:
-|Logging level for Query.
-|`debug`, `info`, `warn`, `error`, `fatal`, `panic`
+|`value` (inside component)
+|Value is the taint value the toleration matches to. If the operator is Exists, the value should be empty, otherwise just a regular string.
+|type: string
 |
 
-|options:
-  query:
-    base-path:
-|You can set the base path for all tempo-query HTTP routes to a non-root value: for example, `/tempo` will cause all UI URLs to start with `/tempo`. This can be useful when running `tempo-query` behind a reverse proxy.
-|`/<path>`
+|`jaegerQuery`
+|JaegerQuerySpec defines Jaeger Query specific options.
 |
+|
+
+|`enabled`
```

Comment:
This is a property of `jaegerQuery`

Reply:
Like this?
https://github.com/openshift/openshift-docs/pull/66373/files#r1441549271